test/python/WMCore_t/BossAir_t/PyCondorPlugin_t.py

```diff
@@ -456,6 +456,23 @@ def testT_updateJobInfo(self):
 
         return
 
+    def test_CMSGroupsRegex(self):
+        """
+        _test_CMSGroupsRegex_
+
+        Simply test the regular expression for the CMSGroups classad
+        """
+        mapGroupReqName = {'HIRun2015': 'jen_a_ACDC_HIRun2015-HIMinimumBias5-02May2016_758p4_160721_015926_2101',
+                           'HIG': 'pdmvserv_HIG-RunIISummer15GS-01331_00274_v2__160308_111346_296',
+                           'EXO': 'pdmvserv_task_EXO-RunIISpring16DR80-03516__v1_T_160719_140341_8564',
+                           'BPH': 'pdmvserv_BPH-Summer12-00208_00263_v1__160726_025423_731',
+                           'undefined': 'amaltaro_TaskChain_MC_DMWM_Test_July_Patches_160727_234956_6253'}
+
+        for group, request in mapGroupReqName.items():
```

Comment:
Yes!! I always use that, don't know why skipped it here. Updating the code asap, thanks!